### PR TITLE
Create and remove books

### PR DESCRIPTION
**Pull request by Chimwemwe-127001**
# Create and remove books
> In this feature update:
- [x] Configure the Redux Store.
- [x] Write the book's actions and reducer
- [x] Use the `redux-logger` to check if the application is working correctly

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Chimwemwe-127001 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddNewBook.js [hunk 1]
```diff
@@ -1,18 +1,56 @@
-import React from 'react';
+/* eslint-disable consistent-return */
```
Comment (mikenath223) [inline src/components/AddNewBook.js]: - [x] Kindly remove this eslint disable rule. Its best to fix the issue in the code.

File: src/components/AddNewBook.js [hunk 2]
```diff
@@ -1,18 +1,56 @@
...
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+// import your Action Creators
```
Comment (mikenath223) [inline src/components/AddNewBook.js]: - [x] Please remove this commented code.

File: src/index.js [hunk 3]
```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/configureStore';
 import Bookstore from './components/Bookstore';
 // import { Provider } from 'react-redux';
```
Comment (mikenath223) [inline src/index.js]: - [x] Please remove this commented code.

File: src/components/AddNewBook.js (hunk 1 above)
Comment (Chimwemwe-127001) [reply src/components/AddNewBook.js]: Done

File: src/index.js (hunk 3 above)
Comment (Chimwemwe-127001) [reply src/index.js]: Done

File: src/components/AddNewBook.js (hunk 2 above)
Comment (Chimwemwe-127001) [reply src/components/AddNewBook.js]: Done

**Comment by Chimwemwe-127001** [comment]
- Hello @mikenath223 . I have made changes with reference to the above code review. Hopefully, my PR gets approved this time.
  - Happy coddiong :computer: :smiley:

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Chimwemwe-127001, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**